backend/public/app/src/behaviors/csync-behavior.html

```diff
@@ -63,7 +95,14 @@
         let errorCallback = function (err) {
           console.error(err)
         }
-        this.csyncApp.authenticate("facebook", facebookAccessToken).then(successCallback, errorCallback)
+        let localSuccessCallback = function () {
+          this.set("csyncAuthSuccess", true)
+          successCallback()
+        }.bind(this)
+        this.csyncApp.authenticate("facebook", facebookAccessToken).then(localSuccessCallback, errorCallback)
+        this.csyncApp.authenticate("demo", "demoToken(" + facebookAccessToken + ")").then(localSuccessCallback, errorCallback)
```

Comment:
whoa. the demo one is not supposed to be committed. just made a commit with a fix.